map/map_tests/transliteration_test.cpp

```diff
@@ -0,0 +1,36 @@
+#include "testing/testing.hpp"
+
+#include "coding/multilang_utf8_string.hpp"
+#include "coding/transliteration.hpp"
+
+#include "platform/platform.hpp"
+
+// This test is inside of the map_tests because it uses Platform for obtaining the resource directory.
+UNIT_TEST(Transliteration_CompareSamples)
+{
+  Transliteration & translit = Transliteration::GetInstance();
```

Comment:
coding_tests project doesn't allow to use Platform:
`DEFINES += OMIM_UNIT_TEST_DISABLE_PLATFORM_INIT`